packages/web-frontend/src/containers/MapOverlayBar/TitleAndDatePicker.js

```diff
@@ -1,60 +1,77 @@
 import React, { useEffect, useState } from 'react';
+import MuiBox from '@material-ui/core/Box';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import moment from 'moment';
 import { withStyles } from '@material-ui/core/styles';
-import DownArrow from '@material-ui/icons/ArrowDropDown';
 import MuiSwitch from '@material-ui/core/Switch';
 import CircularProgress from '@material-ui/core/CircularProgress';
 import { DateRangePicker } from '../../components/DateRangePicker';
 import { getDefaultDates, getLimits, GRANULARITY_CONFIG } from '../../utils/periodGranularities';
 import { Content, ContentText } from './Content';
-import { TUPAIA_ORANGE } from '../../styles';
+import { MAP_OVERLAY_SELECTOR, TUPAIA_ORANGE } from '../../styles';
 import { setDisplayedMapOverlays } from '../../actions';
+import { Pin } from './Pin';
+
+const FlexSpaceBetween = styled(MuiBox)`
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-start;
+`;
 
 const Switch = withStyles({
```

Comment:
We try to use styled-components wherever possible for consistency. You should be able to style this using styled-components by using the mui classnames to target the sub elements. eg. "MuiButtonBase-root" and "MuiSwitch-thumb"

Reply:
Agree and thanks for requesting so I can try to use `styled-component` again in here. Before I tried to use `.MuiSwitch-root` for `root` but it didn't work. This time I found we need to specify css for `MuiSwitch-root` in the base.